packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -330,6 +331,7 @@ export class SeenGossipBlockInput {
           resolveAvailability(blockData);
           // TODO(das): should not use syncUnknownBlock metrics here
           metrics?.syncUnknownBlock.resolveAvailabilitySource.inc({source});
+          metrics?.dataColumns.bySource.inc({source: blockData.dataColumnsSource}, blockData.dataColumns.length);
```

Comment:
Lets just do a single increment here because this will get called with each column that gets received.

```suggestion
          metrics?.dataColumns.bySource.inc({source: blockData.dataColumnsSource});
```

Reply:
The source here will only be `GOSSIP`